client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import {
+  useTranslation,
+  DetailViewSkeleton,
+  AlertModal,
+  useNavigate,
+  RouteBuilder,
+  useDebounceCallback,
+} from '@openmsupply-client/common';
+import { EncounterFragment, useEncounter } from '../api';
+
+import { useJsonForms } from '../../Patient/JsonForms';
+import { AppRoute } from '@openmsupply-client/config';
+import { Toolbar } from './Toolbar';
+
+export const DetailView: FC = () => {
+  const t = useTranslation('patients');
+  const { data: encounter, isLoading } = useEncounter.document.get();
+  const navigate = useNavigate();
+
+  const handleSave = useEncounter.document.upsert(
+    encounter?.patient.id ?? '',
+    '',
+    encounter?.document.documentRegistry?.documentType ?? ''
+  );
+  const { JsonForm /* saveData, isDirty, validationError */ } = useJsonForms(
```

Comment:
this is a little temporary.. have just pulled the json data in for now - but will change to the bubble tabs